youtube_dl/extractor/orf.py

```diff
@@ -223,6 +254,66 @@ class ORFFM4IE(ORFRadioIE):
     }
 
 
+class ORFNOEIE(ORFRadioIE):
+    IE_NAME = 'orf:noe'
+    IE_DESC = 'Radio Niederösterreich'
+    _VALID_URL = r'https?://(?P<station>noe)\.orf\.at/player/(?P<date>[0-9]+)/(?P<show>\w+)'
```

Comment:
Add `only_matching` test for each extractor.

Reply:
Are md5 and info_dict required? All the shows wont be available after 7 days ...
Is the md5 the md5sum of the downloaded file?

Reply:
No, only_matching as already pointed out.

Reply:
Also there will be multiple files downloaded in general ...

Reply:
Again: **ONLY MATCHING** test, no downloading, no metadata, nothing.

Reply:
Oh ok, so not even the url field is supposed to be there?

Reply:
Sry I saw your comments too late.